config/initializers/git_api_client.rb

```diff
@@ -14,4 +14,10 @@
   end
 
   config.cache_store = Rails.cache
+
+  config.circuit_breaker = {
+    enabled: true,
+    threshold: 5,
```

Comment:
What was the rational behind the threshold? I can't remember if we talked about this or not?

Reply:
Number of consecutive errors that will break the circuit

Reply:
Yeah sorry I mean where did we get 5 min 5 minutes from?